### DISCUSSION TITLE
How can I test Venmo flow

### DISCUSSION THREAD

**Discussion by aminezebdi** [braintree/braintree_android#753] [Q&A]
I would like to test a successful deposit flow with Venmo by launching Venmo app. I don't have an account cannot create one. Is there a way to test the whole flow ? I also went through this page https://developer.paypal.com/braintree/docs/guides/venmo/testing-go-live

**Comment by scannillo** [comment]
👋 Hello - this looks like a duplicate message. Please see response [here](https://github.com/braintree/braintree_ios/discussions/1068#discussioncomment-6345223).
